internal/credman/credman_windows_test.go

```diff
@@ -0,0 +1,107 @@
+package credman
+
+import (
+	"testing"
+	"time"
+)
+
+func TestWriteCredential(t *testing.T) {
+	credential := &Credential{
+		TargetName:     "TestTargetName",
+		Comment:        "TestComment",
+		LastWritten:    time.Now(),
+		UserName:       "TestUsername",
+		Persist:        PersistSession,
+		CredentialBlob: []byte{0x65, 0x66},
+	}
+
+	err := WriteCredential(credential, CredTypeGeneric)
+	if err != nil {
+		t.Errorf("WriteCredential returned an error: %v", err)
```

Comment:
>Errorf( [](http://example.com/codeflow?start=4&length=7)

this one could be `Fatalf`, unless you are just making sure that if WriteCredential somehow wrote the credential but still returned an error it will get deleted.

Reply:
This is now a assert.NoErrorf